acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/privacy/PrivateContractPublicStateAcceptanceTest.java

```diff
@@ -169,4 +171,43 @@ public void privateContractMustNotBeAbleToCallSelfDestructOnPublicContract() thr
         .returns(
             "0x", e -> ((PrivateTransactionReceipt) e.getTransactionReceipt().get()).getOutput());
   }
+
+  @Test
```

Comment:
did you verify that this test failed without the fix?

Reply:
Yes, https://github.com/hyperledger/besu/pull/2816/commits/342ee16f956a725af11262f50a66243d21ed1b22 causes the original test to fail which is correct as we are actually trying to send value to a public contract from a private contract. 

Added the additional test as a precaution.